packages/server/tests/acceptance/rateLimiter.spec.ts

```diff
@@ -60,11 +58,18 @@ describe('@ratelimiter Rate Limiters Acceptance Tests', function () {
                     await new Promise(r => setTimeout(r, 1));
                 }
             }
-            const error = predefined.IP_RATE_LIMIT_EXCEEDED(testConstants.ETH_ENDPOINTS.ETH_CHAIN_ID);
 
-            await expect(sendMultipleRequests()).to.eventually.be.rejected.and.satisfy((err) => {
-                return [error.code, error.name, error.message].every(substring => err.body.includes(substring));
-            });
+            // TODO: adapt to the new ethers v6 JsonRpcProvider error handling
```

Comment:
Q: Is this still applicable?
If so please open a ticket to capture this effort next.

Reply:
I already did. It's in the description [here](https://github.com/hashgraph/hedera-json-rpc-relay/issues/1617) but with no labels :confused:.